src/scenes/AudioPuzzleScene.tsx

```
@@ -1,2082 +1,789 @@
+  position: [number, number, number];
   rotation?: [number, number, number];
   scale?: number;
-  cubeColors?: string[]; // Add prop for cube colors
-  syncComplete?: boolean; // Add syncComplete prop
+  color?: string;
 }
 
-const ScifiScreen: React.FC<ScifiScreenProps> = ({
-  position = [0, 0, 0],
-  rotation = [0, 0, 0],
+const GreekUrn: React.FC<UrnProps> = ({ 
```

ESLint

[error] 'GreekUrn' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)